h2o-py/tests/testdir_algos/glm/pyunit_GH_6722_beta_equality_loose_lessthan_linear_constraints_binomial.py

```diff
@@ -0,0 +1,211 @@
+import h2o
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator as glm
+from tests import pyunit_utils
+from tests.pyunit_utils import utils_for_glm_tests
+
```

Comment:
```suggestion


```
Ideally, I would try to stick to PEP-8 and [keep 2 blank lines between top-level functions](https://peps.python.org/pep-0008/#blank-lines).